src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -888,6 +888,7 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
         }
 
         [Fact]
+        [OuterLoop]
```

Comment:
>[OuterLoop] [](start = 8, length = 11)

Can you add a string arg indicating why it's outerloop?  Same goes for other cases.  We need to do a pass through all of the outerloop tests at some point to do so, but we can at least add them for new outerloops moving forward.

Reply:
Sure